app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    """
+    This class represents a cafe.
+    """
```

Comment:
There is no need to add this docstring, remove it. It does not add additional useful information